lib/logger.js

```diff
@@ -0,0 +1,3 @@
+var logger = require('@google/cloud-diagnostics-common').logger;
+var logLevel = process.env.GCLOUD_ERRORS_LOGLEVEL;
```

Comment:
It would be good to support setting this through the configuration object as well. It is fine to open a bug and do that in a follow on.